pkg/ebpf/tracee.go

```diff
@@ -1139,77 +1140,74 @@ func (t *Tracee) populateBPFMaps() error {
 	return nil
 }
 
-// getTailCalls collects all tailcall dependencies from required events, and
-// generates additional tailcall per syscall traced. For syscall tracing, there
-// are 4 different relevant tail calls:
+// getTailCalls collects all tailcall dependencies for the events picked to be traced, and generates
+// additional tailcall per syscall traced. For syscall tracing, there are 4 different relevant tail
```

Comment:
nit: Let's start "For syscall tracing..." on a new line as it is a different subject.